README.md

```diff
@@ -61,7 +61,7 @@ int main()
 
     // Create a task that awaits the doubling of its given value and
     // then returns the result after adding 5.
-    auto double_and_add_5_task = [&](uint64_t input) -> coro::task<uint64_t> {
+    auto double_and_add_5_task = [${EXAMPLE_CORO_TASK_CPP}](uint64_t input) -> coro::task<uint64_t> {
```

Comment:
I'm not sure why your pre-commit is doing this :grimacing: but it looks like it did it again :(